test/tests/functional/pbs_resv_start_dur_end.py

```diff
@@ -140,3 +140,17 @@ def test_end_wall_resv(self):
 
         a = {'reserve_state': (MATCH_RE, 'RESV_CONFIRMED|2')}
         self.server.expect(RESV, a, id=rid)
+
+    def test_rstat_longterm_resv(self):
+        now = int(time.time())
+        a = {'Resource_List.select': '1:ncpus=1',
+             'reserve_start': now + 3600,
+             'reserve_end': now + 4294967296}
```

Comment:
if you expect the start time to 1 hr from now then wouldn't the duration be  "4294967296 - 3600" which is not going to overflow the integer, right?
I suggest you increase the end time by another 3600 seconds.